[2022-10-17 10:48:45,083] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: my_first_dag.process_user backfill__2022-01-14T00:00:00+00:00 [queued]>
[2022-10-17 10:48:45,110] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: my_first_dag.process_user backfill__2022-01-14T00:00:00+00:00 [queued]>
[2022-10-17 10:48:45,110] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-17 10:48:45,110] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-10-17 10:48:45,110] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-17 10:48:45,161] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): process_user> on 2022-01-14 00:00:00+00:00
[2022-10-17 10:48:45,191] {standard_task_runner.py:52} INFO - Started process 4915 to run task
[2022-10-17 10:48:45,200] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'process_user', 'backfill__2022-01-14T00:00:00+00:00', '--job-id', '171', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpzy6k0ily', '--error-file', '/tmp/tmpbwlf3doh']
[2022-10-17 10:48:45,213] {standard_task_runner.py:80} INFO - Job 171: Subtask process_user
[2022-10-17 10:48:45,331] {task_command.py:369} INFO - Running <TaskInstance: my_first_dag.process_user backfill__2022-01-14T00:00:00+00:00 [running]> on host 48b5b9061ccc
[2022-10-17 10:48:45,474] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_first_dag
AIRFLOW_CTX_TASK_ID=process_user
AIRFLOW_CTX_EXECUTION_DATE=2022-01-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=backfill__2022-01-14T00:00:00+00:00
[2022-10-17 10:48:45,506] {python.py:173} INFO - Done. Returned value was: None
[2022-10-17 10:48:45,551] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=process_user, execution_date=20220114T000000, start_date=20221017T104845, end_date=20221017T104845
[2022-10-17 10:48:45,624] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-17 10:48:45,672] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
